e2e/tests/nightly/admin/workspace-admin-ui.spec.ts

```diff
@@ -0,0 +1,143 @@
+    await workspaceAdminPage.getWorkspaceNamespaceInput().type(workspaceNamespace);
+    workspaceAdminPage.clickLoadWorkspaceButton();
+    await workspaceAdminPage.waitForLoad();
+    //verify that the Notebook Preview button is disabled
+    expect(await workspaceAdminPage.getNotebookPreviewButton().isCursorNotAllowed()).toBe(true);
+    const accessReasonText = 'testing purposes';
+    await workspaceAdminPage.getAccessReasonTextArea().type(accessReasonText);
+    //verify that the Notebook Preview button is now enabled
+    expect(await workspaceAdminPage.getNotebookPreviewButton().isCursorNotAllowed()).toBe(false);
+    await workspaceAdminPage.clickNotebookPreviewButton();
+    const adminNotebookPreviewPage = new AdminNotebookPreviewPage(page);
+    await adminNotebookPreviewPage.waitForLoad();
+    // verify that the notebook preview page loaded
+    const previewCode = await adminNotebookPreviewPage.getFormattedCode();
+
+    // verify that the notebook file is in Python programming language
+    expect(previewCode.some((item) => item.includes('import pandas'))).toBe(true);
+    expect(previewCode.some((item) => item.includes('import os'))).toBe(true);
+    expect(await adminNotebookPreviewPage.getNamespaceText()).toEqual(workspaceNamespace);
+    // click on the namespace link to navigate back to the workpsace-admin page
+    await adminNotebookPreviewPage.clickNamespaceLink();
+    await workspaceAdminPage.waitForLoad();
+  });
+
+  test('Verify that admin is able to delete runtime', async () => {
+    const workspaceAdminPage = new WorkspaceAdminPage(page);
+    await workspaceAdminPage.waitForLoad();
+    await new WorkspacesPage(page).load();
+    const workspaceCard = await WorkspaceCard.findCard(page, workspaceName);
+    await workspaceCard.clickWorkspaceName(true);
+    await new WorkspaceDataPage(page).waitForLoad();
+    const runtimePanel = new RuntimePanel(page);
+    await runtimePanel.open();
+    // Create runtime
+    await runtimePanel.createRuntime();
+    await page.waitForTimeout(2000);
+    // navigate back to workspace admin page
+    await navigation.navMenu(page, NavLink.WORKSPACE_ADMIN);
+    await workspaceAdminPage.waitForLoad();
+    await workspaceAdminPage.getWorkspaceNamespaceInput().type(workspaceNamespace);
+    await workspaceAdminPage.clickLoadWorkspaceButton();
+    await workspaceAdminPage.waitForLoad();
+    //verify the runtime status is running
+    expect(await workspaceAdminPage.getRuntimeStatus()).toEqual('Running');
+    const deleteRuntimeModal = await workspaceAdminPage.clickRuntimeDeleteButton();
+    await deleteRuntimeModal.clickCancelButton();
+    await workspaceAdminPage.waitForLoad();
+    await workspaceAdminPage.clickRuntimeDeleteButton();
+    // delete the runtime
+    await deleteRuntimeModal.clickDeleteButton();
+    await page.waitForTimeout(4000);
+
+    //verify the runtime status is deleting
+    expect(await workspaceAdminPage.getRuntimeStatus()).toEqual('Deleting');
```

Comment:
The test is failing here, since it ends before the status changes from "running" to "deleting"

Reply:
right, it times out. I can try increasing the timeout.

Reply:
@jmthibault79 after increasing the timeout all the 3- test cases have passed.